### PR TITLE
chore(package): update most to 0.18.0

### PR DESCRIPTION
**Pull request by TylorS**
Updates dev and regular dependency on most to 0.18.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,10 @@
   "version": "1.0.0",
   "description": "Streamlined DOM Events for most.js",
   "main": "dist/dom-event.js",
+  "typings": "dist/dom-event.d.ts",
```
Comment (briancavalier) [inline package.json]: Let's do only the 0.18.0 version bump first.  We're close on the TS defs for cujojs/most, but I think we can merge this more quickly without trying to include TS here.

Comment (TylorS) [inline package.json]: Oops!!! Didn't checkout from master :-1:

**Comment by TylorS** [comment]
Fixed!

**Comment by briancavalier** [comment]
:metal: thanks!

**Pull request merged**